static/src/javascripts/projects/commercial/modules/third-party-tags.ts

```diff
@@ -37,7 +38,9 @@ const addScripts = (tags) => {
 			if (typeof tag.url !== 'undefined') {
 				script.src = tag.url;
 			}
-			script.onload = tag.onLoad;
+			// script.onload cannot be undefined
+			// TODO change type ThirdPartyTag.onLoad in commercial core
+			script.onload = tag.onLoad ?? null;
```

Comment:
I think this (nullish coalescing) is better than changing the type of `ThirdPartyTag.onLoad`.

Reply:
I agree - I'll remove the comment